pyaedt/modules/MeshIcepak.py

```diff
@@ -8,7 +8,7 @@
 
 
 class IcepakMesh(object):
-    """Manages Icepak meshes.
+    """Class that manages Icepak meshes.
```

Comment:
```suggestion
    """Manages Icepak meshes.
```
Alex told me that he wanted all classes to start with a verb ending in "s". Ignore if the guidelines have been changed.

Reply:
good point. I've applied also to other class descriptions which was not complying with it.